kernel/src/vm/vmar/mod.rs

```diff
@@ -813,6 +813,11 @@ impl<R> Vmar<R> {
     pub fn get_rss_counter(&self, rss_type: RssType) -> usize {
         self.0.get_rss_counter(rss_type)
     }
+
+    /// Returns the virtual memory size in bytes.
+    pub fn get_vm_size(&self) -> usize {
+        self.0.inner.read().total_vm
```

Comment:
Naming it `get_mapped_size` or `get_mappings_total_size` is probably more self-describing than `get_vm_size`. Having a longer time also implies that the method is more expensive (it takes a lock).